data/newMigrations/003-service_categories.js

```diff
@@ -0,0 +1,13 @@
+exports.up = function (knex) {
+    return knex.schema.createTable('service_categories', tbl => {
+        tbl.increments('service_category_id')
+            .primary();
+        tbl.string('service_category_description', 1000)
```

Comment:
For the description you can take out the notNullable, and switch it to a `.text` instead of a `.string`